README.md

```diff
@@ -584,13 +584,11 @@ On Ubuntu using Unix Makefiles
 Most dependencies can be obtained via the Ubuntu software repositories. On each
 line below, we show the corresponding package.
 
-* **operating system**: Ubuntu 13.10 or later.
+* **operating system**: Ubuntu 12.04 or later. *(See OS specific instructions under 'For the impatient' below)*
```

Comment:
This is the last remaining issue for this PR. What do you think about the suggestion, @klshrinidhi? I think it aligns with the idea that there is no minimum operating system version. I don't think we should state "12.04" if we do not yet give detailed instructions for it.